src/logic/embedding.lean

```diff
@@ -15,6 +15,7 @@ universes u v w x
 namespace function
 
 /-- `α ↪ β` is a bundled injective function. -/
+@[nolint has_inhabited_instance] -- depending on cardinalities, an injective function may not exist
 structure embedding (α : Sort*) (β : Sort*) :=
```

Comment:
Does the linter still fire if you provide a specific instance, such as
```lean
instance : inhabited (embedding α α) := { to_fun := id, inj' := ...}
```

Reply:
Yes, it would.  But I don't really see the point in this instance.

Reply:
Sorry - do you mean "yes it would fire", or "yes that would fix the problem".

The point in the instance would be to make `arbitrary (α ↪ α)` work - although having never used `arbitrary`, I can't say if that is useful either.

Reply:
Thanks for clarifying. I suppose there's little point adding it until someone needs it.